tests/braille-specs/malayalam.yaml

```diff
@@ -0,0 +1,75 @@
+﻿display: unicode.dis
+table: malayalam.cti
+
+flags: {testmode: bothDirections}
+tests:
+  - [൦, ⠼⠚]	# MALAYALAM DIGIT ZERO
+  - [൧, ⠼⠁] # MALAYALAM DIGIT ONE
+  - [൨, ⠼⠃]	# MALAYALAM DIGIT TWO
+  - [൩, ⠼⠉]	# MALAYALAM DIGIT THREE
+  - [൪, ⠼⠙]	# MALAYALAM DIGIT FOUR
+  - [൫, ⠼⠑]	# MALAYALAM DIGIT FIVE
+  - [൬, ⠼⠋]	# MALAYALAM DIGIT SIX
+  - [൭, ⠼⠛]	# MALAYALAM DIGIT SEVEN
+  - [൮, ⠼⠓]	# MALAYALAM DIGIT EIGHT
+  - [൯, ⠼⠊]	# MALAYALAM DIGIT NINE
+  - [൰, ⠼⠁⠚]	# MALAYALAM NUMBER TEN
+  - [൱, ⠼⠁⠚⠚]	# MALAYALAM NUMBER ONE HUNDRED
+  - [൲, ⠼⠁⠚⠚⠚]	# MALAYALAM NUMBER ONE THOUSAND
+  - [൳, ⠼⠁⠌⠙]	# MALAYALAM FRACTION ONE QUARTER
+  - [൴, ⠼⠁⠌⠃] # MALAYALAM FRACTION ONE HALF
+  - [൵, ⠼⠉⠌⠙]	# MALAYALAM FRACTION THREE QUARTERS
+
+  - [മർക്കൊസ്, ⠍⠈⠻⠈⠅⠅⠭⠎⠄]
```

Comment:
What exactly do the tests below do? Is it just a list of words? Not that it's so important for me personally to understand the tests because I don't know the Malayalam script anyway, but in general it's just a good idea to add some explanation to tests.

Reply:
@bertfrees 
Thank you for your comments. I will work more on the test as you suggest. Yes it's basically a list of words specifically those that are affected by my changes to the table. I am basically pulling out words from, in the order they appear in, a book of the Bible that I have transcribed and had proof read by a Malayalam proof reader.
I have also contacted Dipendra (named on the original table) via email to see if he can confirm the changes. So it will be good to wait to see if he responds.

Reply:
> I have also contacted Dipendra (named on the original table) via email to see if he can confirm the changes. So it will be good to wait to see if he responds.

Ah yes good idea.